README.md

```diff
@@ -18,6 +18,7 @@ Create a `.env` file in the root with the following values inside it (each varia
 - `OAUTH2_CLIENT_ID`: Client ID of Discord OAuth2 Application (see prerequisites).
 - `OAUTH2_CLIENT_SECRET`: Client Secret of Discord OAuth2 Application (see prerequisites).
 - `ALLOWED_URL`: Allowed origin for CORS middleware.
+- `PRODUCTION`: Set to False if running on localhost. Defaults to true.
```

Comment:
Please set this to false by default in `docker-compose.yml`.

Reply:
This wasn't enough on its own due to the way docker handles external traffic, so I just resorted to allowing all non-production traffic.